src/test/java/calculator/StringCalculator.kt

```diff
@@ -0,0 +1,31 @@
+package calculator
+
+class StringCalculator {
+    fun add(
+        text: String?,
+    ): Int = if (text.isNullOrBlank()) {
+        EMPTY_RESULT
+    } else parseText(text = text)
```

Comment:
저도 명확성때문에 이름 있는 아규먼트 호출을 선호하지만 
파라미터가 한개 인데다가 변수명이 명확해서 아규먼트명을 생략하면 조금 더 간결해 질 것 같은데 재권님 생각은 어떠세요?
```suggestion
    ): Int = if (text.isNullOrBlank()) {
        EMPTY_RESULT
    } else {
        parseText(text)
    }    
```

Reply:
단일 일 때는 생략해도 괜찮을 것 같다고 생각합니다!
[613ef3b](https://github.com/next-step/ddd-legacy/pull/586/commits/613ef3bb35ad0f3fabda64a76fd72e81a7e193c2) 에서 반영 완료 했습니다.